Microsoft.DotNet.Interactive/KernelBase.cs

```diff
@@ -89,25 +88,16 @@ command switch
             loadExtension.Handler = async context =>
             {
                 var assembly = AssemblyLoadContext.Default.LoadFromAssemblyPath(loadExtension.AssemblyFile.FullName);
-
-                var extensionTypes = assembly
-                                     .ExportedTypes
-                                     .Where(t => typeof(IKernelExtension).IsAssignableFrom(t))
-                                     .ToArray();
-
-                foreach (var extensionType in extensionTypes)
-                {
-                    var extension = (IKernelExtension) Activator.CreateInstance(extensionType);
-
-                    await extension.OnLoadAsync(pipelineContext.Kernel);
-                }
+                await ExtensionLoader.LoadFromAssembly(assembly, pipelineContext.Kernel);
```

Comment:
There's a bit of confusion in these methods right now but this should use `context` rather than `pipelineContext` (though they're likely the same instance).

Reply:
Done